simpleimage/make_rootfs.sh

```diff
@@ -144,6 +169,10 @@ exit 101
 EOF
 chmod a+x "$DEST/usr/sbin/policy-rc.d"
 
+# Mount dev
+mount --bind /dev $DEST/dev
+mount --bind /dev/pts $DEST/dev/pts
```

Comment:
Without /dev mounted, the script throws errors on my end saying that needed files in /dev are missing (because it is not mounted).